types/prettier/index.d.ts

```diff
@@ -139,6 +139,12 @@ export interface Plugin {
     defaultOptions?: Partial<RequiredOptions>;
 }
 
+export interface BrowserParserPlugin {
```

Comment:
The browser plugin is actually the same as the Node.js plugin (`Plugin`), though it seems we did not describe it well:

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/03186dbc08372aa1ca9689147386523588be6efd/types/prettier/index.d.ts#L134-L140

Their fields should be all optional.

Reply:
Thanks @ikatyang! Updated :ballot_box_with_check: